.github/actions/check_docstrings/action.yml

```diff
@@ -0,0 +1,56 @@
+name: Check Docstrings
+inputs:
+    python-version:
+        description: 'The version of Python to use for the workflow.'
+        default: '3.10'
+        required: false
+        type: string
+    repository:
+        description: 'The repository to check the docstrings for.'
+        required: True
+        type: string
+    branch:
+        description: 'The branch to check the docstrings for.'
+        required: false
+        type: string
+        default: ''
+    package-name:
+        description: 'The name of the package to check the docstrings for.'
+        required: True
+        type: string
+    names-to-skip:
+        description: 'The names of the modules/classes/functions/methods to skip in the docstring check.'
```

Comment:
Can you write an example here in the description to see how the inputs are passed. Are they an string-fied list?